azure/functions/_http_wsgi.py

```diff
@@ -192,13 +191,16 @@ def _handle(self, req, context):
         wsgi_request = WsgiRequest(req, context)
         environ = wsgi_request.to_environ(self._wsgi_error_buffer)
         wsgi_response = WsgiResponse.from_app(self._app, environ)
-        self._handle_errors()
+        self._handle_errors(wsgi_response)
         return wsgi_response.to_func_response()
 
-    def _handle_errors(self):
+    def _handle_errors(self, wsgi_response):
```

Comment:
As we are passing the wsgi_response, are there other potential things to look at, apart from status code to help in the error reporting?

Reply:
checked dont see other fields in wsgi_response can help with error reporting